DQMOffline/JetMET/python/jetAnalyzer_cfi.py

```diff
@@ -164,12 +164,20 @@
 
 jetDQMAnalyzerAk8PFPUPPICleanedMiniAOD=jetDQMAnalyzerAk4PFCHSCleanedMiniAOD.clone(
     jetsrc = cms.InputTag("slimmedJetsAK8"),
```

Comment:
@ramankhurana  shouldn't there also be a line "JetType = cms.string('miniaod')," here?

Reply:
@ahinzmann  Yes it should be there, I see the link from your PR:

https://github.com/ahinzmann/cmssw/blob/fa8a49d9e88f4b9d3b570f935eda196593105d7a/DQMOffline/JetMET/python/jetAnalyzer_cfi.py

that this line is already there. I am not sure what is the confusion/problem. Please let me know.

Reply:
ok, yes, it is there